services/repository/src/main/java/org/sagebionetworks/repo/web/UrlHelpers.java

```diff
@@ -1127,7 +1127,19 @@
 	public static final String OAUTH_2_USER_INFO = AUTH_OAUTH_2+"/userinfo";
 	public static final String OAUTH_2_JWKS = AUTH_OAUTH_2+"/jwks";
 	public static final String OAUTH_2_AUTH_REQUEST_DESCRIPTION = AUTH_OAUTH_2+"/description";
-	
+	public static final String OAUTH_2_REVOKE = AUTH_OAUTH_2+"/revoke";
```

Comment:
do other oauth providers publish their revocation endpoint in their discovery document and, if so, should we do it too?